roles/nginxplus/tasks/conf/upload-config.yml

```diff
@@ -44,35 +53,27 @@
   when: nginx_template_upload_enable
   tags: update_conf
 
-- name: "Setup: Ensure NGINX Stream Directory Exists"
-  file:
-    path: "{{ nginx_stream_upload_dest }}"
-    state: directory
-    mode: 0755
-  when: nginx_stream_upload_enable
-  tags: update_conf
-
-- name: "Setup: Upload NGINX Stream Configuration Files"
-  copy:
-    src: "{{ item }}"
-    dest: "{{ nginx_stream_upload_dest }}"
+- name: Setup | Upload NGINX HTTP Templated Site Config Files
+  ansible.builtin.template:
+    src: http/library.conf.j2
+    dest: /tmp/{{ item.name }}.conf
+    # dest: /etc/nginx/conf.d/{{ item.name }}.conf
```

Comment:
Should we put in a comment explaining that this is where they will eventually go, once testing is complete, or is that self-explanatory?